pkgs/applications/networking/instant-messengers/skype/default.nix

```diff
@@ -56,7 +56,7 @@ stdenv.mkDerivation rec {
 
     # Fixup desktop file
     substituteInPlace skype.desktop --replace \
-      "Icon=skype.png" "Icon=$out/libexec/skype/icons/SkypeBlue_48x48.png"
+      "Icon=skype.png" "Icon=$out/libexec/skype/icons/SkypeBlue_128x128.png"
```

Comment:
The freedesktop way of doing things is to have something like `Icon=skype`, have all icons in
a freedesktop theme (e.g.: hicolor) under the same name using the appropriate freedesktop icon them file hierarchy and let the system find the most appropriate icon depending on the context.